src/game/save_version.h

```diff
@@ -50,7 +51,8 @@ typedef enum {
     SCENARIO_LAST_NO_DYNAMIC_OBJECTS = 8,
     SCENARIO_LAST_NO_EXTENDED_REQUESTS = 9,
     SCENARIO_LAST_NO_EVENTS = 10,
-    SCENARIO_LAST_NO_CUSTOM_MESSAGES = 11
+    SCENARIO_LAST_NO_CUSTOM_MESSAGES = 11,
+    SCENARIO_LAST_NO_CART_DEPOT = 12
```

Comment:
I don't think we need to bump the scenario version as that one doesn't handle cart depots.

Reply:
Done.